auraed/src/graceful_shutdown.rs

```diff
@@ -77,15 +86,25 @@ impl GracefulShutdown {
         // wait for all subscribers to drop
         self.shutdown_broadcaster.closed().await;
 
+        if let Err(e) = self.cell_service.stop_all().await {
```

Comment:
This seemed out of order. We should try and stop before freeing, I'd imagine?

Reply:
good catch.